Directory.Build.targets

```diff
@@ -34,4 +49,32 @@
     <PackageReference Include="Microsoft.DotNet.XUnitExtensions" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(IsSourceProject)' == 'true'">
+    <AssemblyAttribute Include="System.CLSCompliantAttribute" Condition="'$(CLSCompliant)' == 'true'">
+      <_Parameter1>true</_Parameter1>
+    </AssemblyAttribute>
+    <AssemblyAttribute Include="System.Reflection.AssemblyConfigurationAttribute">
+      <_Parameter1>$(Configuration)</_Parameter1>
+    </AssemblyAttribute>
```

Comment:
We don't want the AssemblyConfigurationAttribute:

```suggestion
```

Reply:
I'm confused. In the comment above you said:

>> [assembly: AssemblyConfiguration("Release")]

>> [assembly: NeutralResourcesLanguage("en-US")]

> These two properties are missing in this repo:

I thought you were trying to say that we need to add them in this repo.

Reply:
Oooh ok, you were referring to the property only. Ok. Removing the metadata.